### PR TITLE
Fix #6672 When adding attribute with ref entity to table, the complete table breaks

### LINKED ISSUE #6672: When adding attribute with ref entity to table, the complete table breaks
### How to Reproduce
Upload 
[vkgl-model.xlsx](https://github.com/molgenis/molgenis/files/1317714/vkgl-model.xlsx)
Go to 'Attribute', in the dataexplorer
Add a new attribute (categorical mref) to the 'consensus'
Delete the new attribute
### Expected behavior
Add a new attribute (categorical mref): Works fine
Delete the new attribute: Deletes fine

### Observed behavior
Add a new attribute (categorical mref): Complete table broken
Delete the new attribute: Weird postgresql error, but when just clicking once and when being patient, the attribute gets deleted (and the table works again phew!)

### PR DESCRIPTION
**Pull request by Mark-de-Haan**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [x] Clean commits

### CODE REVIEW

File: molgenis-metadata-manager/src/main/frontend/src/store/actions.js [hunk 1]
```diff
@@ -103,7 +102,8 @@ export default {
    */
   [GET_ATTRIBUTE_TYPES] ({commit}: { commit: Function }) {
     api.get('/api/v2/sys_md_Attribute/meta/type').then(response => {
-      commit(SET_ATTRIBUTE_TYPES, response.enumOptions.map((type) => type.toUpperCase()))
+      // Workaround for differences between Categorical MREF enum on the server and in the REST api
+      commit(SET_ATTRIBUTE_TYPES, response.enumOptions.map(type => type === 'categoricalmref' ? 'CATEGORICAL_MREF' : type.toUpperCase()))
```
Comment (ConnorStroomberg) [inline molgenis-metadata-manager/src/main/frontend/src/store/actions.js]: Could you add a test case and explain the 'difference between Categorical MREF and ...'

**Comment by ConnorStroomberg** [comment]
Seems to work, did you add a test case to the test plan for this case, or is it a edge case ?

After the fix there is no error adding a attribute via the metadata editor,  but do we also need to fix this in the backend ? The error report talks about; 'Weird postgresql error' . 

Also when i add a attribute and click the save button, it takes several seconds ( on my local machine ) for the success message to appear ( indexing ? ). During this time the user is not informed about what is going on.

**Comment by ConnorStroomberg** [comment]
@fdlk  suggest the [issue concerning the reference entity not being show](https://github.com/molgenis/molgenis/issues/6736) in fixed as part of: https://github.com/molgenis/molgenis/pull/6674/files 

I will test the fix and close this PR if it is no longer needed

**Comment by ConnorStroomberg** [comment]
@sidohaakma fixed part of this issue in 

https://github.com/molgenis/molgenis/pull/6674/files

I have send a path to @fdlk with the updated fixes to the frontend. 

After both of these have been merged the issue should be fixed.
